pywbemtools/pywbemcli/_cmd_instance.py

```diff
@@ -113,6 +115,14 @@
                       'supported. '
                       'Default: No properties modified.')]
 
+keybinding_key_option = [              # pylint: disable=invalid-name
+    click.option('-k', '--key', type=str, metavar='KEYNAME=VALUE',
+                 required=False, multiple=True,
+                 help='Value for a key in keybinding of CIMInstanceName. '
+                      'May be specified multiple times. '
+                      'Allows defining keys without the issues of quotes. '
+                      'Default: No initial properties provided.')]
```

Comment:
On "No initial properties provided": These are neither initial nor general properties. Suggest to change this to "No keybindings provided".

Reply:
DONE